src/ContentRepository/ApplicationModel/ApplicationStorage.cs

```diff
@@ -304,7 +305,7 @@ private static AppNode LoadApps(out List<string> appNames, out List<Application>
             using (new SystemAccount())
             {
                 //var result = nq.Execute();
-                var result = ContentQuery.Query(SafeQueries.TypeIs, QuerySettings.AdminSettings, typeof(Application).Name);
+                var result = ContentQuery_NEW.Query(SafeQueries.TypeIs, QuerySettings.AdminSettings, typeof(Application).Name);

```

Comment:
Unorthodox names indicate that they elements are semifinished and mean strong intention that they will be refactored.